src/services/dumpers/templates/agent-nodejs/index.hbs

```diff
@@ -0,0 +1,68 @@
+require('dotenv').config();
+const { createAgent } = require('@forestadmin/agent');
+{{datasourceImport}}
+{{#unless isMongoose}}
+
+const dialectOptions = {};
+
+{{#if (or isMySQL isMariaDB)}}
```

Comment:
Is this still needed with agent v2?

Reply:
I checked the agent's code, and it's not handled internaly. The SSL part is required for sure if we want to be ISO v1.